projects/Ownft.md

```diff
@@ -0,0 +1,100 @@
+
+Several alternative solutions exist, such as dedicated 3rd party pinning services and Filecoin-backed incentivization systems. 
+
+We developed an easy to use self-hostable IPFS-backed software stack, that runs on a low-power, cheaply available computing device - the Raspberry Pi. 
+
+With this stack, NFT collectors have guaranteed data availability, provided by the locally running IPFS node. They also get a personalized view into their collection. The stack is geared towards Zora-based NFTs, so collectors are able to seamlessly participate in the Zora marketplace from the same self-owned UI. 
+
+## Use of IPFS, Filecoin and Libp2p
+<!-- Describe how your project uses any or all of these technologies, and why. -->
+
+The project uses go-ipfs for ARM processors to run a local IPFS node on a Raspberry Pi. Then, a web server written in Node.JS serves an endpoint that the UI uses to fetch NFT tokens belonging to the user, and automatically pins hashes found in the contentURI fields. 
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+
+This project is currently under development and in the MVP stage.
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+
+|       |       |
+|------------|-------------|
+| <img src="https://imgur.com/NlA8083.png" width="250"> | <img src="https://imgur.com/voXX6Fo.png" width="250"> |
+| <img src="https://imgur.com/DSei9Mu.png" width="250"> | <img src="https://imgur.com/7zZoiYL.png" width="250"> |
+
+## Target Audience
+<!--Describe who will be your project's users-->
+
+- NFT collectors
+- User's who want to self-host their NFT content
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+
+Our hardware + software stack is currently under development so we do not have any users
+
+## Github repo
+<!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+
+https://github.com/sha256rma/ownft_v2
+
+## Website
+<!--Link your website if available-->
+
+The application is still under development so the website is not hosted publicly at the moment
+
+## Docs
+<!--Including a link to your project docs!-->
+
+https://github.com/sha256rma/ownft_v2/blob/master/README.md
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
+### Team Size
+
+4
+
+### Team members
+
+Kartikeya
+- full-stack developer
+- computer science graduate
+- https://github.com/sha256rma
+
+Taehun
+- back-end developer / business
+- entrepreneurship major graduate
+- https://github.com/kevtae
+
+Denny
+- front-end developer
+- computer science student
+- https://github.com/denny999222
+
+Amit
+- front-end developer
+- Information Technology major student
+- https://github.com/akumar1999
+
+
+## How the community can engage
+* Email: ksharma@dype.market
```

Comment:
Please see the requirements - you need to make a discussion thread in this repo's Discussions section and link it here.

Reply:
https://github.com/ipfs/community/discussions/550